client/Packages/com.beamable/Editor/UI/Content/Components/ContentVisualElement/ContentVisualElement.cs

```diff
@@ -223,12 +224,13 @@ private void NameLabel_OnKeydown(KeyDownEvent evt)
 
 		private void NameLabel_OnKeyup(KeyUpEvent evt)
 		{
+			evt.StopPropagation();
 			CheckName();
 		}
 
 		private void NameLabel_OnBlur(BlurEvent evt)
 		{
-			BeamEditorContext.Default.Dispatcher.Schedule(CommitName);
+			evt.StopPropagation();

```

Comment:
does content renaming still work, though? If I click on a content name field, change the text, and then click on something else in Unity- how does the name get saved?

Reply:
Yes, the visualization in the content manager and the scriptable object of that content are both changed. Is there somewhere else that I should also check?

Reply:
hmm, okay that seems good enough